### PR TITLE
Add docs for new Sensirion SHT31 sensor

### PR DESCRIPTION
**Pull request by viorels**
**Description:**

Add support for Sensirion SHT31 sensor

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#12952

## Checklist:

- [X] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.sht31.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+sensor:
+  platform: sht31
+  name: Bedroom
+  i2c_address: 0x44
+  monitored_conditions:
+    - temperature
+    - humidity
+```
+
+Configuration variables:
```
Comment (OttoWinter) [inline source/_components/sensor.sht31.markdown]: Some time ago we switched to using a template for configuration variables: https://home-assistant.io/developers/documentation/standards/#component-and-platform-pages Some docs (too many 😕) still use the old way, but for new integrations we should definitely use the new style. 

```markdown
{% configuration %}
  name:
    description: The name of the sensor. Temperature and Humidity will be added to the name for the sensor name.
    required: false
    default: SHT31
    type: string
  i2c_address:
    description: I2C address of the sensor.
    required: false
    default: "`0x44`"
    type: int
  monitored_conditions:
    description: Conditions to monitor.
    required: false
    default: All conditions
    type: list
    keys:
      temperature: 
        description: The current temperature of the SHT31.
      humidity:
        description: The current humidity of the SHT31.
{% endconfiguration %}
```
------
Also: in your code, `monitored_conditions` is optional and defaults to all conditions. The documentations are wrong here.

File: source/_components/sensor.sht31.markdown [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+title: "Sensirion SHT31 Sensor"
+description: "Instructions how to integrate SHT31 sensors within Home Assistant."
+date: 2018-03-06 19:15
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: DIY
+ha_release: N/A
+logo: dht.png
```
Comment (OttoWinter) [inline source/_components/sensor.sht31.markdown]: I don't think this is a DHT sensor 😬

Comment (viorels) [reply source/_components/sensor.sht31.markdown]: Other than this markdown file and a new image, is there anything else I should add for a new brand/sensor? Thanks

**Comment by frenck** [comment]
Rebased the PR in order to fix the build

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks, @viorels! Looks good 👍 

Can be merged as soon as the parent PR gets merged.

**Review by frenck — APPROVED** [review]

**Pull request merged**